Extend guess range outside of data range in Muon analysis (#33108)
mantidproject/mantid: docs/source/release/v6.3.0/muon.rst

- The plots no longer use scientific notation for the axis values.
- On resizing priority is given to plotting.
- The Sequentially Fit all button is now visible for 4K displays

> - the plot guess option in Fitting can now have its range interpolated or extrapolated.

Reviewer: 'The' is missing a capital letter

> - the plot guess option in Fitting can now have its range extended beyond the range of the data.

Reviewer 2: ```suggestion
- the plot guess option in Fitting can now have its range interpolated or extrapolated. 
```
